streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableFilterTest.java

```diff
@@ -222,25 +233,30 @@ private void doTestNotSendingOldValue(final StreamsBuilder builder,
 
             final List<MockProcessor<String, Integer>> processors = supplier.capturedProcessors(2);
 
-            processors.get(0).checkAndClearProcessResult("A:(1<-null) (ts: 5)", "B:(1<-null) (ts: 10)", "C:(1<-null) (ts: 15)");
-            processors.get(1).checkAndClearProcessResult("A:(null<-null) (ts: 5)", "B:(null<-null) (ts: 10)", "C:(null<-null) (ts: 15)");
-
+            processors.get(0).checkAndClearProcessResult(new KeyValueTimestamp<>("A", "(1<-null)", 5),
```

Comment:
I suspect the test failures in this class are due to the fact the value here is a String `"(1<-null)"` for the expected value but what is returned from processing is a `Change` object so the test fails.

For example the expected values are created like
```java
new KeyValueTimestamp<>("B", "(1<-null)", 10)
```
but should be 
```java
new KeyValueTimestamp<>("B", new Change(1, null), 10)
```
 I suspect the issue is the same in other failures as well.

Reply:
agree, will update.